doc/mysqlnd_azure_log.md

```diff
@@ -0,0 +1,124 @@
+2 | Print logs to a pre-defined file by mysqlnd\_azure's config mysqlnd\_azure.logfilePath=xx.log. | [MYSQLND_AZURE] [ERROR] CLIENT_SSL is not set when mysqlnd_azure.enableRedirect is ON
+4 | Print logs using stderr. <br/> - PHP CLI: Print to console. <br/> - Work scenario with Apache/Nginx: Will be discussed in seperated section below. | [MYSQLND_AZURE] [ERROR] CLIENT_SSL is not set when mysqlnd_azure.enableRedirect is ON
+
+#### Detailed behavior for logOutput1 and logOutput4
+- **For mod-php and Apache (take php7.2 as example)**
+
+Config file | Config options | Usage
+:----- | :------ | :----- 
+/etc/php/7.2/apache2/php.ini | error_log <br/> log_error <br/> display_errors   | This file is Apache mod_php's config file s <br/> -log_error: logOutput1 is only available if this is on. <br/> error_log: where logOutput1 goes into. <br/> --1.	error_log=syslog:  logOutput1 is not available with this option. <br/> --2.	error_log=A_file_path. It will take priority of apache’s error log setting. <br/> display_errors: Whether print logOutput1 to a client web page.
+/etc/apache2/sites-available/000-default.conf | ErrorLog | This file is Apache's error log path for seperated virtual instance. <br/> If error_log is not set in php.ini above, logOutput1 will go here.
+/etc/apache2/apache2.conf | ErrorLog | This file is Apache’s global error log path. <br/> logOutput4 goes here. <br/> If ErrorLog is not set for each virtual host, and error_log is not set in php.ini, then logOutput1 also goes here.
+ 
+ - **For php-fpm and Apache (take php7.2 as example)**
+ 
+ Config file | Config options | Usage
+:----- | :------ | :----- 
+/etc/php/7.2/fpm/php-fpm.conf | error_log	| This Php-fpm’s system error log, has content like php-fpm is restarted or crashed. <br/> -- logOutput4 will go here if catch_workers_output = yes in www.conf. <br/> logOutput1 will also go here if catch_workers_output = yes in www.conf && log_errors=ON && error_log is not set in php.ini and www.conf below.
+/etc/php/7.2/fpm/php.ini | error_log <br/> log_error <br/> display_errors	| - error_log: The destination logOutput1 goes to. <br/> - log_error: same with mod_php. <br/> - display_errors: same with mod_php.
+/etc/php/7.2/fpm/pool.d/www.conf	| catch_workers_output <br/> php_admin_value[error_log] <br/> php_admin_flag[log_errors] <br/> php_flag[display_errors] | - catch_workers_output: <br/> Whether catch logOutput4 and logOutput1. <br/> <br/> -- php_admin_value[error_log]: <br/> -- php_admin_flag[log_errors]: <br> -- php_flag[display_errors]: <br/> Same as that in php.ini, but the definitions here take priority.
+/etc/apache2/sites-available/000-default.conf	| ErrorLog	| logOutput1 will go here if the error_log is not set in php.ini and www.conf. <br/> Please notice that the logs for single request is in single line since fastcgi treat one request’s log as single line)
```

Comment:
```suggestion
/etc/apache2/sites-available/000-default.conf	| ErrorLog	| logOutput=1 will output here if the error_log is not set in php.ini and www.conf. <br/> Please notice that the logs for single request is in single line since FastCGI treats one request’s log as single line)
```